content/influxdb/cloud/upgrade/v1-to-cloud/_index.md

```diff
@@ -0,0 +1,259 @@
+---
+title: Upgrade from InfluxDB 1.x to InfluxDB Cloud
+description: >
+  To upgrade from InfluxDB 1.x to InfluxDB Cloud, migrate data, and then create
+  database and retention policy (DBRP) mappings.
+menu:
+  influxdb_cloud:
+    parent: Upgrade to Cloud
+    name: 1.x to Cloud
+weight: 11
+related:
+  - /influxdb/cloud/upgrade/v1-to-cloud/migrate-cqs/
+---
+
+To upgrade from **InfluxDB OSS 1.x** to **InfluxDB Cloud**:
+
+1. [Create an InfluxDB Cloud account](#create-an-influxdb-cloud-account)
+2. [Create an All-Access authentication token](#create-an-all-access-authentication-token)
+3. [Download and install the `influx` CLI](#download-and-install-the-influx-cli)
+4. [Create DBRP mappings](#create-dbrp-mappings)
+5. [Dual write to InfluxDB 1.x and InfluxDB Cloud](#dual-write-to-influxdb-1x-and-influxdb-cloud)
+6. [Migrate time series data](#migrate-time-series-data)
+7. [Migrate continuous queries](#migrate-continuous-queries)
+8. [Collaborate with other users](#collaborate-with-other-users)
+
+## Create an InfluxDB Cloud account
+Do one of the following to create an InfluxDB Cloud account:
+
+- [Subscribe through InfluxData](/influxdb/cloud/get-started/#subscribe-through-influxdata) and
+  [start for free](/influxdb/cloud/get-started/#start-for-free).
+- [Subscribe through your cloud provider](/influxdb/cloud/get-started/#subscribe-through-a-cloud-provider).
+
+## Create an All-Access authentication token
+InfluxDB Cloud requires all requests to be authenticated with **token authentication**.
+Create an **All-Access** token in your InfluxDB Cloud user interface (UI) to use
+for the upgrade process.
+
+1. Click **Data (Load Data) > Tokens** in the left navigation bar.
+
+    {{< nav-icon "data" >}}
+2. Click **{{< icon "plus" >}} Generate**, and then select **All-Access Token**.
+3. Enter a description for the token, and then click **{{< icon "check" >}} Save**.
+
+_For more information about managing tokens and token types, see [Manage tokens](/influxdb/cloud/security/tokens/)._
+
+## Download and install the influx CLI
+Visit the [InfluxDB downloads page](https://portal.influxdata.com/downloads/)
+and download the **InfluxDB Cloud CLI** (`influx`).
+Place the `influx` binary in your system `PATH` or execute the CLI commands from
```

Comment:
```suggestion
2. Place the `influx` binary in your system `PATH` or execute the CLI commands from
```

Reply:
Would it be helpful to provide a command to put the binary in your PATH here? I feel like we've done this somewhere else?

Reply:
This process and the `PATH` location is OS-specific and I didn't want to introduce more OS complexity here.